src/backend/base/langflow/load/load.py

```diff
@@ -106,17 +106,11 @@ def run_flow_from_json(
     Returns:
         List[RunOutputs]: A list of RunOutputs objects representing the results of running the flow.
     """
-    # Set all streaming to false
-    try:
-        import nest_asyncio
-
-        nest_asyncio.apply()
-    except Exception:  # noqa: BLE001
-        logger.opt(exception=True).warning("Could not apply nest_asyncio")
     if tweaks is None:
         tweaks = {}
     tweaks["stream"] = False
-    graph = load_flow_from_json(
+    graph = asyncio.to_thread(
```

Comment:
Why put `load_from_json` in the thread instead of `run_graph`?

Reply:
`run_graph` calls ˋarun` so it’s better to make it a coroutine. Since it’s a coroutine, no need to put it in a thread.